packages/dev/docs/pages/react-spectrum/theming.mdx

```diff
@@ -0,0 +1,107 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import docs from 'docs:@react-types/provider/src/index.d.ts';
+import {Layout, TypeLink} from '@react-spectrum/docs';
+export default Layout;
+
+```tsx import
+import {Provider} from '@react-spectrum/provider';
+import {ActionButton} from '@react-spectrum/button';
+```
+
+---
+category: Concepts
+---
+
+# Theming
+
+This page describes how themes works in React Spectrum, including how applications adapt
+to operating system dark mode and touch scale.
+
+## Introduction
+
+React Spectrum is built to support theming. Colors, sizing, and spacing options can be customized
+through the use of CSS variables which are defined using the [Provider](Provider.html) component. Themes
+consist of definitions for these variables for light and dark [color schemes](https://spectrum.adobe.com/page/color/#Color-themes-and-modes),
+along with medium and large [platform scales](https://spectrum.adobe.com/page/platform-scale/).
```

Comment:
Maybe `Themes consist of definitions for color schemes, variables light and dark, and platform scales, variables medium and large.` My concern was about the `for ... for` with the previous sentence being my attempt to rectify that.

Reply:
Also do light, dark, medium, and large all need to be formatted with back ticks?